backend/app/routes/Q&A/answers/approve.js

```diff
@@ -0,0 +1,18 @@
+const answer = require('../../../models/answers');
+
+module.exports = (req, res, next) => {
+  try {
+    let answerId = req.body.answerId; //getting answer id from body
+    answer.findOne({ _id: answerId }).then((answer) => {
+      if (!answer) {
+        return res.status(404).json({ error: 'answer id not found' });
+      }
+      answer.isApproved = true;
```

Comment:
Let's take the value of `isApproved` in request body as adim can approve & disApprove both